app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,14 @@
         }
       });
     </script>
+
+  <!-- Google Tag Manager -->
```

Comment:
I'd rather this was in a separate file - I think I saw you ask how-to on slack?

The "asset pipeline" guide is a good read for how you might do this with Rails.

[This section](https://guides.rubyonrails.org/asset_pipeline.html#asset-organization) explains how you might include a vendor javascript file and how they're imported, and [this section](https://guides.rubyonrails.org/asset_pipeline.html#coding-links-to-assets) explains how you'd then reference them in `.erb` files.